pkg/secrethub/http.go

```diff
@@ -121,6 +123,13 @@ func newHTTPClient(authenticator auth.Authenticator, opts *ClientOptions) *httpC
 	}
 }
 
+func (c *httpClient) Authenticate(in interface{}) (*api.Session, error) {
+	var out api.Session
+	rawURL := fmt.Sprintf(pathAuthenticate, c.base)
+	err := c.post(rawURL, http.StatusOK, in, &out)
```

Comment:
Should this expect `http.StatusOK` or `http.StatusCreated`?

Reply:
`http.StatusCreated` sounds good.